app/src/main/java/com/example/harvestfresh/ProductListingAdapter.java

```diff
@@ -30,10 +36,28 @@ public ProductListingAdapter(Context context, List<ProductListing> products) {
     @NonNull
     @Override
     public ProductListingAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-        View view = LayoutInflater.from(context).inflate(R.layout.item_product, parent, false);
+        View view = LayoutInflater.from(context)
+                .inflate(R.layout.item_product, 
+                        parent, false);
```

Comment:
Multiline one, multiline all: move "false" down one. This helps make sure you dont miss a parameter when youre reading through the list later.